rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -89,11 +115,29 @@ void COCOMetaDataReader::print_map_contents()
         bb_coords = elem.second->get_bb_cords();
         bb_labels = elem.second->get_bb_labels();
         img_size = elem.second->get_img_size();
+        mask_cords = elem.second->get_mask_cords();
+        polygon_size = elem.second->get_polygon_count();
+        vertices_count = elem.second->get_vertices_count();
         std::cout << "<wxh, num of bboxes>: " << img_size.w << " X " << img_size.h << " , " << bb_coords.size() << std::endl;
         for (unsigned int i = 0; i < bb_coords.size(); i++)
         {
             std::cout << " l : " << bb_coords[i].l << " t: :" << bb_coords[i].t << " r : " << bb_coords[i].r << " b: :" << bb_coords[i].b << "Label Id : " << bb_labels[i] << std::endl;
         }
+        if (_mask)
+        {
+            int count = 0;
+            std::cout << "\nNumber of objects : " << bb_coords.size() << std::endl;
```

Comment:
Check if the print statements are necessary.

Reply:
Remove them if not

Reply:
This function is added for printing the map contents which contains the COCO metadata so the print statements are necessary